hack/release/operator/make_chart.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
```

Comment:
We have a Make target doing the build too, I'd say this script should read the versions from the toml file and call that target, some changes will be needed, reading from the env var instead of the manager olm file.

Reply:
I am not 100% sure I understand what you are saying. But I can say I didn't want to change the behavior of existing make targets as they are also used in other workflows. You have to either do logic to around the existences of the toml file, or duplicate the code. I went with duplicate till it is proven they are the same.

Reply:
makes sense, if you want I could do the change in my following PR or we could look at it together